packages/office-ui-fabric-react/src/components/ScreenReaderAlert/ScreenReaderAlertPage.tsx

```diff
@@ -0,0 +1,121 @@
+              code={ ScreenReaderAlertRepeatExampleCode as string }
+            >
+              <ScreenReaderAlertRepeatExample />
+            </ExampleCard>
+            <ExampleCard
+              title='ScreenReaderAlert with Spinner'
+              code={ ScreenReaderAlertSpinnerExampleCode as string }
+            >
+              <ScreenReaderAlertSpinnerExample />
+            </ExampleCard>
+            <ExampleCard
+              title='ScreenReaderAlert polite mode'
+              code={ ScreenReaderAlertAfterExampleCode as string }
+            >
+              <ScreenReaderAlertAfterExample />
+            </ExampleCard>
+          </div>
+        }
+        propertiesTables={
+          <PropertiesTableSet
+            sources={ [
+              require<string>('!raw-loader!office-ui-fabric-react/src/components/ScreenReaderAlert/ScreenReaderAlert.Props.ts')
+            ] }
+          />
+        }
+        overview={
+          <div>
+            <p>
+              ScreenReaderAlerts are used to read out the notifications on the page to the blind people with the help of screen readers like JAWS, Chrome VOX and NVDA. It is implemented by live region and overcomes some known issues about live region when using with React.
+            </p>
+            <p>
+              It renders an addition element for screen reader only, so using this component while the text/ui is visible will duplicate the text in the DOM. You should use vanilla live region attributes instead if you don't want this duplicate element.
+            </p>
+            <p>
+              This component has been tested through following combination of screen readers and browsers:
+            </p>
+            <ul>
+              <li>
+                1. <strong>Narrator in windows 10 Enterprise 14393.577</strong> with <strong>Edge 38.14393.0.0</strong>.
+              </li>
+              <li>
+                2. <strong>NVDA 2016.4</strong> with <strong>FireFox v50.1.0</strong>.
+              </li>
+              <li>
+                3. <strong>Chrome VOX v53</strong> with <strong>Chrome v54</strong>.
+                <br />
+                Note: ReadingMode.ReadAfterOtherContent option is currently not supported in ChromeVOX, text will be read immediately.
+              </li>
+              <li>
+                4. <strong>JAWS 18</strong> with <strong>IE 11</strong>.
+              </li>
+            </ul>
+
+            <p>
+              To try the demo in this page, you should at least using one kind of screen reader. E.g. JAWS, ChromeVOX, NVDA.
+              Better use the browser recommended by the screen reader you're using. E.g. IE11 with JAWS, Chrome with ChromeVOX, FireFox with NVDA.
+            </p>
+          </div>
+        }
+        bestPractices={
+          <div></div>
+        }
+        dos={
+          <div>
+            <ul>
+              <li>Use if it would be hard for blind people to know dynamic changes on the page.</li>
```

Comment:
I'm thrown off here on general usability and examples of when to use this. 

The docs you have say to use it where it would be hard for blind people to know dynamic changes on the page, but isn't that subjective? Are there concrete examples you could provide where this should be used?

You provided "notify user the status change of async tasks" which makes a little more sense to me; a long file copy or upload could be a use case, or a "new mails" notification, but would we expect users to manually use this component, or would we just use this in the notification bubble?

And on the flip side, for negative use cases, it would be nice to list concrete examples. For example, @micahgodbolt read through this PR and suggested in a comment to use it for Dropdown and TextField, but I think that would be a really bad idea... maybe for error messages? But even then I don't think this component would be used for that. I'm not sure. Talking with @cschlechty it sounds like the use case for this is very limited to specific scenarios. Could you explicitly call those out?